modules/javafx.web/src/main/native/Tools/DumpRenderTree/java/DumpRenderTree.cpp

```diff
@@ -135,6 +135,26 @@ JNIEXPORT jboolean JNICALL Java_com_sun_javafx_webkit_drt_DumpRenderTree_shouldS
     return bool_to_jbool(gTestRunner->shouldStayOnPageAfterHandlingBeforeUnload());
 }
 
+JNIEXPORT jobjectArray JNICALL Java_com_sun_javafx_webkit_drt_DumpRenderTree_openPanelFiles
+    (JNIEnv* env, jclass)
+{
+    ASSERT(gTestRunner);
+    jobjectArray ret = (jobjectArray)env->NewObjectArray(gTestRunner->openPanelFiles().size(),
+                        env->FindClass("java/lang/String"), env->NewStringUTF(""));
+    int idx = 0;
+    for (auto i : gTestRunner->openPanelFiles()) {
```

Comment:
Rename `i` to `file`. Since we need index, I feel the traditional for loop would suit better in this place.
```
const auto& openFiles = gTestRunner->openPanelFiles();
static auto& stringCls = env->FindClass("java/lang/String");
jobjectArray files = env->NewObjectArray(openFiles.size(),  stringCls, env->NewStringUTF(""));
for (size_t i  = 0; i < openFiles.size(); i++) {
    env->SetObjectArrayElement(files, i, env->NewStringUTF(openFiles[i].c_str()));
}
return files;
```